### PR TITLE
refactor: distill to pretty

### PR DESCRIPTION
**Pull request by tsao-chi**
fix https://github.com/aya-prover/aya-dev/issues/828

### CODE REVIEW

**Comment by ice1000** [comment]
I believe that you need a different package name. The `pretty` module already occupied the `org.aya.pretty` package, or did it?

**Comment by ice1000** [comment]
Yeah I think it did.

**Comment by ice1000** [comment]
What about this: `org.aya.prettier`?

If you have a better idea, feel free to use it 😉

**Comment by tsao-chi** [comment]
> What about this: `org.aya.prettier`?


ok

File: base/src/main/java/org/aya/concrete/desugar/BinExprParser.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.aya.concrete.Expr;
 import org.aya.concrete.error.OperatorError;
 import org.aya.generic.Constants;
-import org.aya.pretty.doc.Doc;
+import org.aya.prettier.doc.Doc;
 import org.aya.ref.DefVar;
 import org.aya.resolve.ResolveInfo;
```
Comment (imkiva) [inline base/src/main/java/org/aya/concrete/desugar/BinExprParser.java:11]: Please don't change `org.aya.pretty` in the pretty module.

**Review by imkiva — CHANGES_REQUESTED** [review]
Still possible package name clashes: "org.aya.prettier" exist in both "pretty" and "base"

File: base/src/main/java/org/aya/concrete/desugar/BinExprParser.java (hunk 1 above)
Comment (tsao-chi) [reply base/src/main/java/org/aya/concrete/desugar/BinExprParser.java:11]: oh

**Review by imkiva — APPROVED** [review]
bors d=tsao-chi

**Comment by ice1000** [comment]
Thank you!!

**Pull request merged**